modules/passkey/test/userStories/ExecuteWithPaymaster.spec.ts

```diff
@@ -0,0 +1,398 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import { WebAuthnCredentials, decodePublicKey, encodeWebAuthnSignature } from '../utils/webauthn'
+import { buildSafeUserOpTransaction, buildPackedUserOperationFromSafeUserOperation } from '@safe-global/safe-4337/src/utils/userOp'
+import { buildSignatureBytes } from '@safe-global/safe-4337/src/utils/execution'
+import { VerifyingPaymaster } from '../../typechain-types'
```

Comment:
This can be removed, if the type is not required as mentioned in the other comments.